File: data/code_pairs/fortran/1700.f90
program test
  implicit none
  integer :: i
  integer, dimension(10) :: d

  d = 0 ! Initialize all elements to 0
  do concurrent (i=1:5)
     d(i) = i ! Set the first five elements to their 1-based index
  end do

  ! Print the array elements
  do i=1,10
     print *, d(i)
  end do
end program test

File: data/code_pairs/cpp/1700.cpp
#include <iostream>
using namespace std;

int main() {
    int d[10] = {}; // Initialize all elements to 0

    for (int i = 1; i <= 5; ++i) {
        d[i-1] = i; // Arrays in C++ are 0-indexed; set the first five elements to their 1-based index
    }

    // Print the array elements
    for(int i = 0; i < 10; ++i) {
        cout << d[i] << " ";
    }
    cout << endl;
    
    return 0;
}